WebGoatCore/Views/Shared/_CartPartial.cshtml

```diff
@@ -1,24 +1,24 @@
 ﻿@model Cart
 
-<table class="cartTable">
+<table class="receipt">
     <tbody>
-        <tr>
-            <th>Product Id</th>
-            <th>Product Name</th>
-            <th>Unit Price</th>
-            <th>Quantity</th>
-            <th>Extended Price</th>
+        <tr class="headerRow">
+            <th class="tabHeader">Product Id</th>
+            <th class="tabHeader">Product Name</th>
+            <th class="tabHeader">Unit Price</th>
+            <th class="tabHeader">Quantity</th>
+            <th class="tabHeader">Extended Price</th>
         </tr>
         @foreach (var order in Model.OrderDetails.Values)
         {
             <tr class="itemRow">
-                <td class="productId">@order.ProductId</td>
-                <td>@order.Product.ProductName</td>
-                <td>@order.DecimalUnitPrice</td>
-                <td>
+                <td class="tabCell">@order.ProductId</td>
+                <td class="tabCell">@order.Product.ProductName</td>
+                <td class="tabCell">@order.DecimalUnitPrice.ToString("F")</td>
+                <td class="tabCell">
                     <input type="text" value=@order.Quantity class="quantityInput">
                 </td>
-                <td class="extendedPrice">@order.ExtendedPrice</td>
+                <td class="tabCell">@order.ExtendedPrice.ToString("F")</td>
```

Comment:
This looks suspicious. Previously the class defined this cell as special cell (with extended price). I suppose the intention was to format it in a special way (maybe in bold, maybe with larger font). This change makes it a normal cell. (Plus, as mentioned in other comment, "td" is already "table cell")

Reply:
there was no such style defined in css / done